src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -642,7 +643,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			this._defaultKernel = notebookConstants.sqlKernelSpec;
 			this._providerId = SQL_NOTEBOOK_PROVIDER;
 		}
-		if (!this._defaultLanguageInfo || this._defaultLanguageInfo.name) {
+		if (!this._defaultLanguageInfo || !this._defaultLanguageInfo.name) {
```

Comment:
Is this fixing a bug then?

Also can shorten to `!this._defaultLanguageInfo?.name`

Reply:
Yes, @chlafreniere can you double check that my change was correct?